app/Jobs/TranscodeVideo.php

```diff
@@ -159,6 +153,7 @@ protected function transcodeVideo(): void
 
         $this->localDisk->delete($this->tempOriginalFilename);
         $this->localDisk->deleteDirectory($this->getFfmpegTempDirectory());
+        $this->localDisk->deleteDirectory($this->version->Media->User->name);
```

Comment:
same.

Reply:
I'm now only deleting the temp derivatives folder for the current version.